react-native/react-native_3.md

```diff
@@ -0,0 +1,136 @@
+# Making your app with React Native components
+
+As explained in the first part, *React Native* exposes the native components of the platform (iOS or Android). Even if some components can be platform specific, most components are available for both OSes.
+
+## Some React Native basic components
+
+We're first going to have a quick overview of React Native most useful components.
+
+### View
+
+A view is the basic building block. It could be compared to a HTML `div`.
+
+### Text
+
+You already saw it in the last chapter: it's the primitive block used to display text in your app.
+
+```js
+<Text>Hello mobile app world</Text>
+```
+
+Styling is applied to most components using the `style` attribute:
+
+```js
+<Text style={{fontSize: 14, color: 'blue'}}>Hello mobile app world</Text>
+```
```

Comment:
Mention that unlike in css, here the parameters you can pass as style are different for a text element than a view element. Also redirect to the doc about styling for text